src/bufr_interface.h

```diff
@@ -17,151 +17,158 @@
+   @param dim_2 dimensionality of data to read or write.
+   @param iret return value, length of data read.
+   @param table_b_mnemonic string of mnemonics.
 
-    @param[in] bufr_unit - int: the fortran file unit number to read from
-    @param[inout] c_data - void**: c style pointer to a pre-allocated buffer
-    @param[in] dim_1, dim_2 - int: dimensionality of data to read or write
-    @param[out] iret -  int: return value, length of data read
-    @param[in] table_b_mnemonic - const char*: string of mnemonics
+   @author Ronald Mclaren @date 2020-07-29
 */
   void ufbint_f(int bufr_unit, void** c_data, int dim_1, int dim_2,
                 int* iret, const char* table_b_mnemonic);
 
 
-/** @author Ronald McLaren
-    @date 2020-07-29
+/**
+   Wraps BUFRLIB ufbrep() function.
 
-    @brief Wraps BUFRLIB "ufbrep" function.
+   @param bufr_unit the fortran file unit number to read from.
+   @param c_data pointer to a pointer to a pre-allocated buffer.
+   @param dim_1 dimensionality of data to read or write.
+   @param dim_2 dimensionality of data to read or write.
+   @param iret length of data read.
+   @param table_b_mnemonic string of mnemonics.
 
-    @param[in] bufr_unit - int: the fortran file unit number to read from
-    @param[inout] c_data - void**: c style pointer to a pre-allocated buffer
-    @param[in] dim_1, dim_2 - int: dimensionality of data to read or write
-    @param[out] iret -  int: return value, length of data read
-    @param[in] table_b_mnemonic - const char*: string of mnemonics
+   @author Ronald Mclaren @date 2020-07-29
 */
   void ufbrep_f(int bufr_unit, void** c_data, int dim_1, int dim_2,
                 int* iret, const char* table_b_mnemonic);
 
 
-/** @author Ronald McLaren
-    @date 2021-02-24
+/**
+   Wraps BUFRLIB mtinfo() function.
 
-    @brief Wraps BUFRLIB "mtinfo" function.
+   @param path the path where the WMO tables are stored.
+   @param file_unit_1 number to use for first file unit.
+   @param file_unit_2 number to use for second file unit.
 
-    @param[in] path - const char*: the path where the WMO tables are stored
-    @param[in] file_unit_1 - int: number to use for first file unit
-    @param[in] file_unit_2 - int: number to use for second file unit
+   @author Ronald Mclaren @date 2020-07-29
 */
   void mtinfo_f(const char* path, int file_unit_1, int file_unit_2);
 
 
   // Table Data
-/** @author Ronald McLaren
-    @date 2022-03-23
+/**
+   Wraps BUFRLIB status() function.
 
-    @brief Wraps BUFRLIB "status" function.
+   @param file_unit the fortran file unit number to read from.
+   @param lun pointer for the file stream.
+   @param il file status.
+   @param im message status.
 
-    @param[in] file_unit - int: the fortran file unit number to read from
-    @param[out] lun - int: pointer for the file stream
-    @param[out] il - int: file status
-    @param[out] im - int: message status
+   @author Ronald Mclaren @date 2020-07-29
```

Comment:
Also, in all of these param statements in all of these routines, I think they're a lot more readable if we keep the `-` character separating the variable name from the description.  This definitely makes the HTML more readable, imho.

Reply:
Well I could go either way, but we have already done all the other NCEPLIBS documentation with just a space between the name of the param and the description. And I think that looks fine to.

I think it would be useful if all NCEPLIBS projects follow the same conventions in documentation.

I kind of regret not pulling you into the UFS_UTILS documentation effort, where we worked out all these idioms and where you could have made this point at a time it could apply to every project.

If you feel strongly about it we can keep the hyphens in NCEPLIBS-bufr, in order to keep things moving forward quickly...

Reply:
OK, I have put the hyphens back in. @AlexanderRichert-NOAA note that for NCEPLIBS-bufr we will leave a space-hyphen-space after param names. Like this:

```
/** 
    Get pointer to the moda_usrint INV array.

    @param lun - pointer for the file stream.
    @param inv_ptr - pointer to a pointer to the INV array.
    @param inv_size - size of the INV array.
    
    @author Ronald McLaren @date 2022-03-23
*/

```